spec/ddtrace/contrib/rails/redis_cache_spec.rb

```diff
@@ -40,7 +40,12 @@
   end
 
   let(:cache_store_name) do
-    Gem.loaded_specs['redis-activesupport'] ? 'redis_store' : 'redis_cache_store'
+    if Gem.loaded_specs['redis-activesupport'] \
+       && Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new('5.0')
```

Comment:
`redis_cache_store` was introduced in the minor release `5.2`: https://weblog.rubyonrails.org/2018/4/9/Rails-5-2-0-final/
```suggestion
       && Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new('5.2')
```